internal/manifest/v1beta1/manifest_controller_test.go

```diff
@@ -134,41 +125,37 @@ var _ = Describe(
 			func(
 				givenCondition func(manifest *v1beta1.Manifest) error,
 				expectManifestState func(manifestName string) error,
-				expectedHelmClientCache func(cacheKey string) bool,
 			) {
 				manifest := NewTestManifest("oci")
 				Eventually(givenCondition, standardTimeout, standardInterval).
 					WithArguments(manifest).Should(Succeed())
 				Eventually(expectManifestState, standardTimeout, standardInterval).
 					WithArguments(manifest.GetName()).Should(Succeed())
-				Eventually(expectedHelmClientCache, standardTimeout, standardInterval).
-					WithArguments(internalV1beta1.GenerateCacheKey(manifest.GetLabels()[v1beta1.KymaName],
-						strconv.FormatBool(manifest.Spec.Remote), manifest.GetNamespace())).Should(BeTrue())
 				Eventually(deleteManifestAndVerify(manifest), standardTimeout, standardInterval).Should(Succeed())
 			},
 			Entry(
 				"When Manifest CR contains a valid install OCI image specification, "+
```

Comment:
the expected condition should be changed, we are not testing any helm cache-related issues anymore.